sklearn/preprocessing/data.py

```diff
@@ -1173,6 +1177,11 @@ def _combinations(n_features, degree, interaction_only, include_bias):
                                    for i in range(start, degree + 1))
 
     @property
+    def feature_names_(self):
```

Comment:
I'm not sure if a property is better than a function. Actually, I think we use ``get_feature_names()`` for ``DictVectorizer``, ``CountVectorizer`` and ``Pipeline``. So I guess we should use that here, too.